### ISSUE TITLE
Cookie/Cache issue when deployed on Netlify

### ISSUE THREAD

**Issue by valeriosillari** [stephenkr/nuxt-password-protect#5]
Not a real issue, more a question/curiosity.

I set a small nuxt app with the password protect (by the way, awesome work) for testing purpose.

All is working quite nicely locally.

I've also deployed on some Static Hosting service as Now/Zeit and Netlify.

On Now all is good, working as expected.

On Netlify I deploy the page, all is working fine at first interaction, but if I refresh the page (when logged already in) the page "kick me out" as the cookie is reset.

Anybody has experienced  same issue on Netlify?
I guess it's more related to Netlify cookie/cache logic, but I would like to have a kind of confirmation.

**Comment by stephenkr** [comment]
Hi @valeriosillari thank you for using this module.

I have explored your issue above, it's strange because I have this module on a few websites hosted with both Zeit.sh and Netlify.

Perhaps it is a configuration issue. I have updated the example demo Nuxt website available in the repository.

You can test it out here https://nuxt-password-protect.netlify.com/

Please let me know if it works for you.

Thank you
Stephen.

**Comment by valeriosillari** [comment]
Hi @stephenkr,
thanks for the answer and for this awesome module.

Yep it's weird. I tested again on Netlify and I get the issue again.
It's happening on Chrome and Firefox, also in private mode.
It seems that when I "log in" the app doesn't generate anymore the cookie. Even when deploy new commits.

The only thing I've noted: at the first iteration all was good, but then from the index "logged" page I delete manually the cookie with a chrome extension (edit my cookies) and from this moment on all was weird.

Maybe removing the cookie manually (for testing purpose) is the issue?
This happens only on Netlify, not on Zeit.
And on Netlify I did not set any options or changes.

For testing: here the test repo on github and the deploy website on Netlify

- https://github.com/valeriosillari/nuxt-password-test-valerio

- https://nuxt-password-test-valerio.netlify.com/

Not a big deal, I just wondering if this is happening only on my browser or is a edge case.

Nevertheless I love this module, really nice job :)
